src/blueprint/server.rs

```diff
@@ -40,7 +42,10 @@ pub struct Script {
 #[derive(Clone, Debug)]
 pub enum Http {
     HTTP1,
-    HTTP2 { cert: String, key: String },
+    HTTP2 {
+        cert: Vec<CertificateDer<'static>>,
```

Comment:
Why do we need a Vec?

Reply:
`rustls_pemfile::read` returns a vec and the tls acceptor also expects a vec